Core/Inc/control.h

```diff
@@ -23,6 +26,8 @@
 #define RTD_TRACTIVE_VOLTAGE_OFF 200
 // Turn off flag when tractive voltage > 450
 #define RTD_TRACTIVE_VOLTAGE_ON 4500
+//number of teeth on wheel hub
+#define numTeeth 32;
```

Comment:
```suggestion
#define NUM_TEETH 32;
```
Remember the style guide